### DISCUSSION TITLE
View the stats of all Habits all at once in a single place.

### DISCUSSION THREAD

**Discussion by samuelbushi** [iSoron/uhabits#1203] [Feature Requests]
Would be really nice and helpful if we could see the stats of all habits combined in a single analytic place. If this is easy to be implemented I would really appreciate such a feature!

**Comment by hiqua** [comment]
What would you like to see there? Of all the stats visualizations we have, I can only think of the score one as being relatively straightforward to implement for several habits.

**Reply by samuelbushi** [reply]
The need to see the performance of all daily habits at once would be great, don't you think? 

The user could tell how the general performance of the habits goes weekly monthly etc..

**Reply by hiqua** [reply]
But what metric do you mean by "performance"?

**Reply by arsalan6ix** [reply]
An average percentage of all your active habits, it would be a very basic score but would be nice to have as an overall number to see. We can already see individual habit scores by going into them specifically, but this score would reflect the average overall score of all habits combined. It would be nice to have a daily/weekly/monthly/quarterly/bi-annual/yearly toggle to see those particular numbers if desired by the user.

**Reply by anishbekalkar** [reply]
Adding to this chain. If users may not want this, the feature could be made optional but I think it definitely helps in keeping track of overall progress.

**Comment by veggero** [comment]
I'd also love to see such a view. My suggestion would be:

- Score graph with one line for each habit

- For numerical habits with the same unit, a stacked bar graph of that unit over time

- A bar graph over time of % of habits that reached the daily goal / were completed that day

- A list of the habits with a goal, with a progress bar on how much they have been completed

**Comment by Jjj4ns** [comment]
> Would be really nice and helpful if we could see the stats of all habits combined in a single analytic place. If this is easy to be implemented I would really appreciate such a feature!

☝🏼 Is this likely to happen? I'd like to see an overall score of how I'm tracking on all my goals, then compare each month how I've done, i.e 67% achieved in May, 72% in June, to show if you're making overall progress

**Comment by chendy** [comment]
**Simultaneous Calendar View**
I would like to see a calendar view, where a limited amount of tasks could be shown at the same time.

Lets say you have 4 main tasks to view at the same time. 
Each calendar square could be sub-divided into 4 sub squares. Where colours or text/numbers could show the details for each of the 4 sub-tasks.

**Overlaid Graphs**
multiple line or bar plots could be shown.
For bar plots the colours of the task can be sorted, so the lowest bar is show 'first' and so on. That way one can see all the values for various graph plots
